### ISSUE TITLE
I test the vsgearth, but tiles seems not updated

### ISSUE THREAD

**Issue by wwwronin** [timoore/vsgEarth#2]
I print the function call in  TerrainEngineVOE::createTile and TileReaderVOE::read ,out put as following,
![image](https://github.com/timoore/vsgEarth/assets/6395346/fb8e4fbc-7e57-4e94-aad3-c9441a15e650)
When I change the camera use mouse ,there is no output. I think the createTile and read function is not called ,where is the problem. 
Thanks!

**Comment by timoore** [comment]
What is your .earth file?

**Comment by wwwronin** [comment]
<map name="s1">


	
	
	 <GDALElevation name="s1">
		 <url>../data/asiadem.tif</url> 
	 </GDALElevation>



	<GDALImage name="ReadyMap 15m Imagery">
		<url>../data/asia.tif</url>
	</GDALImage>


</map>

**Comment by wwwronin** [comment]
![image](https://github.com/timoore/vsgEarth/assets/6395346/f38a47f1-3d5d-4ec7-ba96-1c8f345a3d8a)
Here is the window ,image is in red aerea.

**Comment by timoore** [comment]
Right, vsgEarth probably won't do the right thing with elevation data that is not whole-Earth. vsgEarth is not really seeing new development; you might try your .earth file with https://github.com/pelicanmapping/rocky, which is the follow-on project to vsgEarth.

**Comment by wwwronin** [comment]
Ok, will try .Thakns a lot.
